### DISCUSSION TITLE
no models showing in UI

### DISCUSSION THREAD

**Discussion by enorrmann** [open-webui/open-webui#4093] [General]
# Bug Report

## Description

**Bug Summary:**
debian 12
ollama models not showing
default ollama installation
i have a working ollama servet which I can access via terminal and it's working

then I installed 

docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

but no models are selectable

**Comment by enorrmann** [comment]
installed ollama via
curl -fsSL https://ollama.com/install.sh | sh

pgrep ollama returns one process id
1760

**Comment by justinh-rahb** [comment]
Set `OLLAMA_HOST=0.0.0.0`

https://github.com/ollama/ollama/blob/main/docs/faq.md#how-do-i-configure-ollama-server

**Reply by enorrmann** [reply]
thank you !

**Reply by nihalsheikh** [reply]
Thanks, this really worked. Just set User Environment Variable > New > Name= OLLAMA_HOST > Value = 0.0.0.0:11434
